db/seeds/abilities.yml

```diff
@@ -1,52 +1,52 @@
 - name: Participate Generally
   internal_id: unrestricted
-  icon: fas fa-heartbeat
-  description: >
+  icon: heartbeat
+  summary: >
     New user restrictions are removed: No limit on posts, vote and comment anywhere
   post_score_threshold: 0.8  # min. +3/-0
   edit_score_threshold: null
   flag_score_threshold: null
 
 - name: Edit Posts
   internal_id: edit_posts
-  icon: fas fa-pen
-  description: >
+  icon: pen
+  summary: >
     Edit all posts (not only own) and review suggested edits to posts
   post_score_threshold: null
   edit_score_threshold: 0.95   # min. +18/-0
   flag_score_threshold: null
 
 - name: Edit Tags
   internal_id: edit_tags
-  icon: fas fa-tags
-  description: >
+  icon: tags
+  summary: >
     Edit tag metadata (excerpt and wiki) and review suggested edits to tags

```

Comment:
This reminds me: I don't know if we currently have UI for suggesting edits to tag wikis and seeing that there are suggestions pending.  If we don't, could you add an issue for that so we don't lose track of it?  Thx.

Reply:
Added to Trello